numpy/core/fromnumeric.py

```diff
@@ -649,6 +649,14 @@ def transpose(a, axes=None):
     >>> np.transpose(x, (1, 0, 2)).shape
     (2, 1, 3)
 
+    >>> x = np.arange(120).reshape(2, 3, 4, 5)
+    >>> x.shape
+    (2, 3, 4, 5)
+
```

Comment:
These lines aren't really interesting, the shape of the array should be obvious from the line above